index.css

```diff
@@ -0,0 +1,42 @@
+body{
```

Comment:
Think about refactoring css into code by type e.g for a button in cases where it is used for both pages?